packages/lib-content/src/screens/Home/DefaultHome/components/Hero.js

```diff
@@ -0,0 +1,118 @@
+import { Anchor, Box, Heading } from 'grommet'
+import styled from 'styled-components'
+import { SpacedText, ZooniverseLogotype } from '@zooniverse/react-components'
+
+import { useTranslation } from '../../../../translations/i18n.js'
+import { mobileBreakpoint } from '../../../../components/SharedStyledComponents/SharedStyledComponents.js'
+
+const Relative = styled(Box)`
+  position: relative;
+  overflow: hidden;
+  background: ${props => props.theme.global.colors['neutral-1']};
+  height: 60vh;
+`
+
+const VideoContainer = styled(Box)`
+  width: 100%;
+  height: 60vh;
+
+  @media (width <= ${mobileBreakpoint}) {
+    display: none;
+  }
+
+  @media (prefers-reduced-motion) {
+    display: none;
+  }
+`
+
+const MobileHeroImage = styled(Box)`
+  width: 100%;
+  height: 60vh;
+
+  @media (width > ${mobileBreakpoint}) {
+    display: none;
+  }
+`
+
+const HeroCopy = styled(Box)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+`
+
+const StyledHeading = styled(Heading)`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  text-shadow: 1px 1px 4px rgba(0, 0, 0, 0.5);
+  width: min(100%, 45rem); // Same as MaxWidthContent
+`
+
+const StyledLogo = styled(ZooniverseLogotype)`
+  filter: drop-shadow(2px 2px 4px rgba(0, 0, 0, 0.5));
+  min-width: 100%;
+`
+
+const StyledLink = styled(Anchor)`
+  width: 300px;
+  border-radius: 5px;
+  font-size: 0.8rem;
+  padding: 8px 5px;
+  text-align: center;
+  color: black;
+  background-color: ${props => props.theme.global.colors['neutral-2']};
+  font-weight: normal;
+  box-shadow: 1px 1px 8px rgba(0, 0, 0, 0.25);
+
+  &:hover {
+    text-decoration: none;
+  }
+`
+
+export default function Hero() {
+  const { t } = useTranslation()
+
+  return (
+    <Relative width='100%'>
+      <VideoContainer>
+        <video
+          autoPlay
+          loop
+          disablePictureInPicture
+          muted
+          preload='metadata'
+        >
+          <source type='video/webm' src='/assets/home-video.webm' />
+          <source type='video/mp4' src='/assets/home-video.mp4' />
+        </video>
+      </VideoContainer>
+      <MobileHeroImage
+        background={`url(${'/assets/home-video-placeholder.jpg'})`}
+      />
+      <HeroCopy justify='center' align='center' pad='medium'>
+        <StyledHeading level={1} margin={{ bottom: '30px', top: '0' }}>
+          <SpacedText
+            color='neutral-2'
+            size='2.4rem'
+            margin={{ bottom: 'small' }}
+          >
+            {t('Home.DefaultHome.mainHeading')}
```

Comment:
I think the font-size in the Figma designs is larger, 18px.

Reply:
This can follow our default button styling.

Reply:
Yes, sorry just catching up on these PRs. [This design](https://zooniverse.github.io/front-end-monorepo/@zooniverse/react-components/index.html?path=/story/components-primarybutton--default ) is what I'm pulling from, although TBF these haven't been updated in some time. I'd prefer the defaults to look how they do in Figma; rounded corners, match the 16pt font. But this can be done at a later point. My thought was it would be easier, then, to update all iterations if we match this Storybook.

Reply:
Agreed it'd be easiest to use the PrimaryButton component as buttons on this page, however as you mentioned there are updates to that shared component required before it can be used to match the Figma design for the homepage. 

The dev team needs to add rounded corners, adjust the font size, and allow for more than 4 background colors. In the PrimaryButton Storybook, there's a "color" dropdown locked to gold, blue, green, and brand-teal, and the background color is also locked to those 4 options in the component code. The Figma design includes a button with transparent for "Sign In" and `neutral-1` for "Register", so those colors need to be added as valid options to the component. I'll create an Issue.

<img width="451" alt="Screenshot 2024-06-05 at 10 09 57 AM" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/14b218e4-33ce-4456-87ac-d93185ec7b84">


Follow-up question -- Are you okay with adding rounded corners to PrimaryButton and the change applying to all FEM pages including Done & Talk, etc on project Classify pages?

Reply:
I'm going to link to our [Design System](https://www.figma.com/design/HUWCyrjkwgPsGKLXhLGb21/Design-System?m=dev&node-id=0-1&t=o3TgwtXdKz1ABRP3-1) for button styling reference, but happy to work through more details when the future issue is ready. 

> Follow-up question -- Are you okay with adding rounded corners to PrimaryButton and the change applying to all FEM pages including Done & Talk, etc on project Classify pages?

100%